### PR TITLE
Enable pep8 validation

### LINKED ISSUE #76: Enable pep8 validation
Right now pep8 validation is being skipped by tox

### PR DESCRIPTION
**Pull request by texasaggie97**
[X] This contribution adheres to [CONTRIBUTING.md]
### What does this Pull Request accomplish?
* Fixes #76 
* Fixes pep8 validations using a combination of:
  * Globally disabling tests
    * H903: Windows style line endings not allowed in code
     * We are using mostly Windows to develop & test
    * E501: Line length
      * Most code is generated using mako - offending lines are mostly lists of parameters to underlying C DLL
    * W391: Blank line at end of file
      * Most code is generated using mako - difficult to remove blank line at the end of the file in some situations
  * Disabling test per line
  * Fixing offending code
* Enable running flake8 both when running 'make test' and during build on travis
  * This means new violations will break the build

### Why should this Pull Request be merged?
* Make our code consistent with pep8

### What testing has been done?
* Travis & system test

### CODE REVIEW

**Comment by texasaggie97** [comment]
I am iffy about some of the "fixes" I put in to make flake8 happy, so I am open to alternatives

File: build/Makefile [hunk 1]
```diff
@@ -2,6 +2,9 @@
 POSSIBLE_TARGETS := module unit_tests run_unit_tests wheel sdist
 export POSSIBLE_TARGETS
 
+ADDITIONAL_POSSIBLE_TARGETS := test flake8
```
Comment (marcoskirsch) [inline build/Makefile]: Now we have POSSIBLE_TARGETS and ADDITIONAL_POSSIBLE_TARGETS
What is the distinction between them? Seems arbitrary, why even have two variables?

Comment (texasaggie97) [inline build/Makefile]: POSSIBLE_TARGETS are all ran as part of `make all`. ADDITIONAL_POSSIBLE_TARGETS are targets you can do explicitly on the command line but are not ran as part of 'all'.

Comment (marcoskirsch) [inline build/Makefile]: DEFAULT_TARGETS and ADDITIONAL_TARGETS ?

Comment (texasaggie97) [inline build/Makefile]: Changed to DEFAULT_TARGETS and ADDITIONAL_TARGETS.

**Pull request merged**